packages/fast-animation/package.json

```diff
@@ -79,7 +79,7 @@
     "tslint": "^5.3.2",
     "tslint-loader": "^3.5.3",
     "tslint-react": "^3.0.0",
-    "typescript": "^2.3.3",
+    "typescript": "^2.9.2",
```

Comment:
Should this match the others ^3.0.1?

Reply:
Per @nicholasrice, so, the typings in that project need a significant refactor because tsc 3.0 adds/changes some of the typings coming out of the animation api.  A new issue will be created for this.

Reply:
yep - see #796